_pages/security/dynamic-scanning.md

```diff
@@ -5,63 +5,38 @@ parent: Security
-    * This will take a while.
-    * There is a progress bar in the Spider and Active Scan tabs. You can click the <img class="inline" src="../../assets/images/zap_graph.png" alt="small graph icon"/> button in the latter to get more details.
-1. Wait for the "Progress" area to say say "Attack complete".
-1. Open the Alerts tab.
+[Compliance Viewer](https://compliance-viewer.18f.gov) runs ZAP scans regularly for sites that opt in. This [continuous monitoring](../../ato/walkthrough/#step-6--continuously-monitor-the-controls) is a step towards being able to do [continuous compliance](../../ato/types/#continuous-authorization).
 
-You can now click on each alert to see more details.
+See the `New Project?` instructions in Compliance Viewer for how to get it set up.
 
-### Examining the Results
-
-#### The Spider
-
-![spider results](../../assets/images/spider_results.png)
-As configured, the Quick Start Attack scan does not follow links to other domains or subdomains. If your project uses either (for example, you use S3 for assets, or the api is at a different sub domain), you will want to click <img class="inline" src="../../assets/images/zap_options.png" alt="small options icon"/> and update the options to include the domains & subdomains within the scope. There is a guide available for those options [here](https://github.com/zaproxy/zap-core-help/wiki/HelpUiDialogsOptionsSpider).
-
-#### Alerts
-
-![alert results](../../assets/images/alert_results.png)
-The Alerts pane lists all alerts discovered while scanning the site. As described on the [alerts page](../scanning#alerts), the red and orange-flagged alerts must be taken care of before the application can be ATO'd. You have a little more flexibility when dealing with the yellow and blue flags, but all of them must be either corrected or, in the case of false positives, documented.
-
-If you're running the attack against a local server you may see some alerts that you wouldn't see on cloud.gov. Debugging web servers are more 'chatty' about errors than production servers.
-
-### Authentication
-
-Authentication can be handled in several ways; the following links provide guidance on handling the most common forms. If these are insufficient, the authentication method is scriptable.
-
-* <https://github.com/zaproxy/zap-core-help/wiki/HelpStartConceptsAuthentication>
-* <https://github.com/zaproxy/zaproxy/wiki/FAQformauth>
+#### Caveats
 
-#### Manual scan
+* [**There is no way to mark false positives.**](https://github.com/18F/compliance-viewer/issues/57) For ATO purposes, you will need to submit a document linking to your project in Compliance Viewer, and then explain in there if there are any false positives.
+* [**There is no authentication support.**](https://github.com/18F/concourse-compliance-testing/issues/98) For any sites that have authentication, you will need to scan the sites manually, as described below.
```

Comment:
@juliaelman @brittag I can't decide if these are better here, in Compliance Viewer, or with the `New Project?` instructions. Thoughts?